Specification/8._Hardware_design_recommendations.md

```diff
@@ -288,17 +286,17 @@ not including additional delay times of CAN transceivers and other components.
 
 #### Automatic bit rate detection
 
-It is encouraged to implement CAN auto bit rate detection when applicable.
+Designers are encouraged to implement CAN auto bit rate detection when applicable.
 Please refer to the CiA 801 application note for recommended practices.
 
-It should be noted that UAVCAN enables the use of the simple bit time measuring approach,
+UAVCAN enables the use of the simple bit time measuring approach,
 as it is guaranteed that any functioning UAVCAN network will always exchange node status messages,
 which can be expected to be published at a rate no lower than 1 Hz,
 and that contain a suitable alternating bit pattern in the CAN ID field.
 Please refer to the [chapter dedicated to the standard data types](/Specification/6._Application_level_functions)
```

Comment:
This refers to "chapter dedicated to the standard data types" but isn't that chapter 7 ? It links to chapter 6 (app level functions)

Reply:
I think it's better to refer to the sixth chapter here because it provides a more elaborate description of the node status reporting feature. Perhaps we should replace "chapter dedicated to the standard data types" with "chapter dedicated to the application-level functions"?

Reply:
Yes. Will do.